contracts/CToken.sol

```diff
@@ -613,6 +613,11 @@ abstract contract CToken is CTokenInterface, ExponentialNoError, TokenErrorRepor
         // EFFECTS & INTERACTIONS
         // (No safe failures beyond this point)
 
+        /* We write the previously calculated values into storage */
```

Comment:
Nice, you didn't think the explanation was helpful?

Reply:
Added the comments back! Sorry, we missed them while making the changes